pims/tests/test_common.py

```diff
@@ -38,16 +38,14 @@ def _skip_if_no_ImageIO():
 
 
 def _skip_if_no_libtiff():
-    try:
-        import libtiff
-    except ImportError:
+    import pims.tiff_stack
```

Comment:
Minor thing: I'd prefer `from pims.tiff_stack import libtiff_available()` at the top of the `test` file. Same for the other `_skip` functions.